homeassistant/components/thermostat/__init__.py

```diff
@@ -301,22 +306,32 @@ def turn_fan_off(self):
     @property
     def min_temp(self):
         """Return the minimum temperature."""
-        return convert(7, TEMP_CELSIUS, self.unit_of_measurement)
+        unit = self.unit_of_measurement
+        if not unit:
```

Comment:
So - this basically wipes out the heat control component. It has literally no way of setting the units; I'm pretty sure it was just grabbing the unit set for HASS. Options:

Pull unit from hass config
Add attribute to device configuration
Both

I assume we should do both - just want input on it.